src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Shape {
+    double getArea();
+
+    void draw();
```

Comment:
create interfaces AreaCalculator with method `getArea()` and Drawable with `draw()`